### PR TITLE
[FE-19] Update ArangoDB logo in WebUI

### PR DESCRIPTION
**Pull request by pluma4345**
### Scope & Purpose

This PR updates the ArangoDB logo in various parts of the WebUI and improves the loginView code somewhat.

- [ ] :hankey: Bugfix
- [x] :pizza: New feature
- [ ] :fire: Performance improvement
- [x] :hammer: Refactoring/simplification

### Checklist

- [ ] Tests
  - [ ] **Regression tests**
  - [ ] C++ **Unit tests**
  - [ ] **integration tests**
  - [ ] **resilience tests**
- [x] :book: CHANGELOG entry made
- [ ] :books: documentation written (release notes, API changes, ...)
- [ ] Backports
  - [ ] Backport for 3.10: *(Please link PR)*
  - [ ] Backport for 3.9: *(Please link PR)*
  - [ ] Backport for 3.8: *(Please link PR)*

#### Related Information

- [ ] Docs PR: 
- [ ] Enterprise PR:
- [x] ~~GitHub issue~~ / Jira ticket: FE-19
- [ ] Design document:

### CODE REVIEW

File: js/apps/system/_admin/aardvark/APP/frontend/js/views/loginView.js [hunk 1]
```diff
@@ -135,61 +119,59 @@
...
 
-      this.collection.login(username, password, this.loginCallback.bind(this, username, password));
+      this.collection.login(username, password, () => this.loginCallback(username, password));
     },
 
     loginCallback: function (username, password, error) {
-      var self = this;
-
       if (error) {
         if (self.loginCounter === 0) {
```
Comment (hkernbach) [inline js/apps/system/_admin/aardvark/APP/frontend/js/views/loginView.js:135]:
```suggestion
        if (this.loginCounter === 0) {
```

Also needs to be changed now?

File: js/apps/system/_admin/aardvark/APP/frontend/js/views/loginView.js [hunk 2]
```diff
@@ -135,61 +119,59 @@
...
+      this.loggedIn = true;
 
       // get list of allowed dbs
       $.ajax({
         type: "GET",
-        url: url,
-        success: function (permissions) {
+        url: frontendConfig.authenticationEnabled
+          ? arangoHelper.databaseUrl(`/_api/user/${encodeURIComponent(username)}/database`, '_system')
+          : arangoHelper.databaseUrl('/_api/database/user'),
```
Comment (hkernbach) [inline js/apps/system/_admin/aardvark/APP/frontend/js/views/loginView.js:160]: That url build logic could be moved into a helper method as also used here:
- https://github.com/arangodb/arangodb/pull/18226/files#diff-cf85c1e65271ee5d2a8659056f3125557933c993640a3a4a7afee2d96a091ac6R71

**Review by hkernbach — CHANGES_REQUESTED** [review]
One potential issue where self has been used but not defined anymore. The other very minor. Apart from that "self reference" LGTM.

**Review by hkernbach — APPROVED** [review]

**Review by Simran-B — APPROVED** [review]
LGTM

**Review by palashkaria — APPROVED** [review]
LGTM

**Review by hkernbach — APPROVED** [review]

**Review by gnusi — CHANGES_REQUESTED** [review]
I don't think iresearch module should be touched by this PR.

**Review by Dronplane — CHANGES_REQUESTED** [review]
Looks like iresearch module was updated. Is it intentional?

**Review by KVS85 — CHANGES_REQUESTED** [review]
Please update iresearch submodule according to base branch.

**Review by MBkkt — CHANGES_REQUESTED** [review]
iresearch submodule shouldn't be update in this pr.

It's probably merge artifact you can revert this change or make reset soft and commit without it

**Review by Dronplane — APPROVED** [review]
LGTM.

**Review by KVS85 — APPROVED** [review]

**Pull request merged**